Gruntfile.js

```diff
@@ -0,0 +1,30 @@
+module.exports = function( grunt ) {
+
+grunt.initConfig( {
+	pkg: grunt.file.readJSON( "package.json" ),
+	jshint: {
+		files: [ "*.js", "routes/**.js" ],
```

Comment:
these should probably be recursive from base directories like `routes/**/*.js` and `static/**/*.js`